changes/5454.housekeeping

```diff
@@ -0,0 +1 @@
+Finished controller models.
```

Comment:
Probably good to add a `.changed` note warning users that upgrading from the beta to the release will not be possible due to the migration rewrite?

Reply:
Added.